quickcomm/admin.py

```diff
@@ -1,13 +1,148 @@
 from django.contrib import admin
+from django.http import HttpResponseRedirect
+from django.utils.html import format_html
 
 # Register your models here.
 
-from .models import Author, Post, Comment, Follow, Like, RegistrationSettings, Inbox
+from .models import Author, CommentLike, Host, HostAuthenticator, Post, Comment, Follow, Like, RegistrationSettings, Inbox
 
-admin.site.register(Author)
+# admin.site.register(Author)
 admin.site.register(Post)
 admin.site.register(Comment)
 admin.site.register(Follow)
 admin.site.register(Like)
 admin.site.register(Inbox)
 admin.site.register(RegistrationSettings)
+admin.site.register(CommentLike)
+
+@admin.register(HostAuthenticator)
+class HostAuthenticatorAdmin(admin.ModelAdmin):
+    list_display = ('username', 'nickname')
+    readonly_fields = ('base64',)
+    actions_on_top = True
+
+    @admin.display(description='Base64 Encoded Username:Password')
+    def base64(self, obj: HostAuthenticator):
+        return obj.base64string
+
+@admin.register(Author)
+class AuthorAdmin(admin.ModelAdmin):
```

Comment:
Is AuthorAdmin just a local author on our server? And AuthorInlineForHost for authors that belong to a HostAdmin (which I assume is some external host)?